.github/workflows/reusable-test.yml

```diff
@@ -160,20 +170,29 @@ jobs:
         shell: bash
         run: |
           yarn workspace @asap-hub/contentful space:create-environment --timeout 600000
+          yarn contentful:migration:run:crn
```

Comment:
If a PR (such as this one) includes migrations then they need to run before any integration tests because the environment clones from Development which won't have those tests.

In this case, without this line then all the integration tests for research outputs fail because research outputs don't exist.